Integrations/Cloaken/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+-
```

Comment:
add a proper note here instead of the dash. describe what you added in the integration
```suggestion
Added new command ***cloaken-screenshot-url***.
```